requirements/base.txt

```diff
@@ -177,7 +177,7 @@ prison==0.2.1
     # via flask-appbuilder
 prompt-toolkit==3.0.28
     # via click-repl
-pyarrow==6.0.1
+pyarrow>=9.0.0, <11 # Warning! python 3.11 requires pyarrow==10.0.1 but some testing libs require pyarrow<10
```

Comment:
I don't think the lock file should have ranges. Let's pin this to the latest 9.x version, and maybe document somewhere that Python 3.11 requires installing >= 10.0.1? What were the libs that require PyArrow < 10?

Reply:
I fix it in next PR when google guys will release 1.0.5 for `db-dtypes`
https://github.com/googleapis/python-db-dtypes-pandas/issues/161

Reply:
@betodealmeida @zhaoyongjie What do you think about it?

Reply:
@EugeneTorap let's wait for 1.0.5 to be released, the Pandas team tend to release fairly frequently.